server/server.go

```diff
@@ -1202,7 +1202,11 @@ func (s *Server) Start() {
 	if opts.JetStream {
 		var cfg *JetStreamConfig
 		if opts.StoreDir != "" {
-			cfg = &JetStreamConfig{StoreDir: opts.StoreDir}
+			cfg = &JetStreamConfig{
+				StoreDir:  opts.StoreDir,
+				MaxMemory: opts.JetStreamMaxMemory,
```

Comment:
I dynamically picked these before, especially memory. Do these get properly set if I define just the store?

Reply:
Yeah I checked it will retain behavior where you dynamically set them if unset here

Reply:
So they would be just set to zero and that triggers the dynamic logic, correct?

Reply:
yup

Reply:
```
server_name: "jetstream"

jetstream {
  store_dir: "/tmp/nats-server"
}
```

```
% ./nats-server --config ~/js.conf
[6862] 2020/01/31 11:49:44.584548 [INF] ----------- JETSTREAM (Beta) -----------
[6862] 2020/01/31 11:49:44.584566 [INF]   Max Memory:      6.00 GB
[6862] 2020/01/31 11:49:44.584573 [INF]   Max Storage:     1.00 TB
[6862] 2020/01/31 11:49:44.584581 [INF]   Store Directory: "/tmp/nats-server/jetstream"
[6862] 2020/01/31 11:49:44.584594 [INF] ----------------------------------------
```